specs/trust-store-trust-policy.md

```diff
@@ -301,6 +388,29 @@ Signature verification levels provide defined behavior for each validation e.g.
     }
 ```
 
+#### Timestamp Countersignature Verification details
+
+ Sufficient and necessary conditions to trigger timestamp countersignature verification:
+  - At least one certificate in the signing certificate chain has expired at the time of verification.
+  - The signing scheme is [`notary.x509`](./signing-scheme.md/#notaryx509).
+  - The unsigned attribute `Timestamp Signature` is present in the signature envelope.
+  - The `authenticTimestamp` under `signatureVerification` of trust policy is NOT marked as `skip`.
+  - `level` under `timestampVerification` of trust policy is set to `strict` or `timestampVerification` is not present, default values are used in this case.
```

Comment:
Do we have a table for levels for `timestampVerification`?

Reply:
The detailed explanation is here: https://github.com/Two-Hearts/specifications/blob/tsa/specs/trust-store-trust-policy.md#version-20

Since it's more like a boolean switch (with only two values `strict` and `skip`). I feel like we don't need an extra table for it?